src/javascripts/application.js

```diff
@@ -37,7 +37,7 @@ nodeListForEach($tabs, function ($tabs) {
 OptionsTable.init()
 
 // Add copy to clipboard to code blocks inside tab containers
-var $codeBlocks = document.querySelectorAll('[data-module="app-copy"]')
+var $codeBlocks = document.querySelectorAll('[data-module="app-copy"] pre')
```

Comment:
We now render code through Markdown so I've moved `data-module` to the wrapper, otherwise every code block on every page will get an unintentional **Copy code** button